### DISCUSSION TITLE
Issue where Orca Slicer 2.2 (release) overrides Bambu Studio opening files from web

### DISCUSSION THREAD

**Discussion by sfarkhan** [OrcaSlicer/OrcaSlicer#7335] [General]
I'm on a Mac and I had Orca 2.2 Beta and Bambu Studio - I could use Bambu on Makerworld to open projects. Once I updated Orca to 2.2 release, it borked Bambu and overrides this so Orca is the default from any web browser - This is the way Chrome/Firefox/Safari. The issue isn't opening files from the OS but how orca slicer is the default for web.

Reinstalling Bambu doesn't fix it. Removing Orca corrects this. I can set the defaults on the OS level so clicking on a 3MF or STL works but not from web links. There was no issue with 2.2 Beta coexisting with Bambu and opening files from links on the web.

**Comment by sfarkhan** [comment]
I spoke to someone and there are supposed to be settings to govern this in Orca but I am not seeing them. His settings vs mine: 
![issue-where-orca-slicer-2-2-release-overrides-bambu-studio-v0-4zp9syg2fdyd1](https://github.com/user-attachments/assets/5960ff39-c0de-46fb-a7ca-f66a787b5307)
![Screenshot 2024-11-01 at 7 13 28 PM](https://github.com/user-attachments/assets/d5c82c47-13ec-44a9-95b3-dbcb34a665a0)

**Comment by sfarkhan** [comment]
I selected Developer Mode and nothing

**Comment by acbaldwi** [comment]
Can confirm same issue here

**Comment by accipiter76** [comment]
Same issue here. It is really annoying.
